releasenotes/notes/spawn-serverless-trace-mini-agent-5c91a600d9c51441.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Adds support for traces and trace metrics in Google Cloud Function environments when used in conjunction with 
```

Comment:
public facing documentation will be added once the serverless trace mini agent package is published to PyPi

Reply:
removed release note